README.md

```diff
@@ -192,22 +164,22 @@ For detailed instructions, see [building](doc/BUILDING.md).
 > OpenAssetIO is a project used in business-critical workflows, and due
 > to this need to work stably and reliably, it is a project with rigid
 > standards.
->
-> Combined with its inherently abstract nature, it is a difficult project
-> for casual contribution. We are a small team, and welcome anyone
-> who can directly help contribute to the core tasks outlined in our
-> [roadmap](ROADMAP.md). We are however, at present, unable to
-> provide "good first tasks" or a gentle, guided, on-ramp for those new
-> to the problem space.
->
-> If you have a prevailing interest in the project, please reach out and
-> start a thread in the Discussions section. This helps us coordinate
-> work and approach to ensure we maintain the standards our users demand
-> of the project. We encourage you to [work with the TSC](https://github.com/OpenAssetIO/OpenAssetIO-WG)
-> before making any big changes or implementing new features.
-
-- See the [contribution guide](doc/contributing/PROCESS.md)
-- Join our [working group](https://github.com/OpenAssetIO/OpenAssetIO-WG) meetings
+
+Combined with its inherently abstract nature, it can be a difficult
+project for casual contribution. We are a small team, and welcome
+anyone who can directly help contribute to the core tasks outlined in
+our [roadmap](ROADMAP.md).
+
+If you have a prevailing interest in the project, please
+[reach out](SUPPORT.md). This helps us coordinate work and approach
```

Comment:
I wonder if we could drop this sentence, as it's somewhat repetitive of earlier paragraphs.

So delete ```This helps us coordinate...demand of the project.```